modules/certmanager/certmanager.tf

```diff
@@ -0,0 +1,62 @@
+resource "null_resource" "certmanager_crds" {
+  triggers = {
+    certmanager_crd_version = var.certmanager_crd_version
+  }
+
+  provisioner "local-exec" {
+    command = "kubectl apply --context ${var.kube_context} --validate=false -f https://raw.githubusercontent.com/jetstack/cert-manager/release-${var.certmanager_crd_version}/deploy/manifests/00-crds.yaml"
+  }
+
+  provisioner "local-exec" {
+    when    = destroy
+    command = "kubectl delete --context ${var.kube_context} -f https://raw.githubusercontent.com/jetstack/cert-manager/release-${var.certmanager_crd_version}/deploy/manifests/00-crds.yaml"
+  }
+}
+
+resource "helm_release" "certmanager" {
+  depends_on = [null_resource.certmanager_crds]
+
+  name       = "cert-manager"
+  repository = "amoy"
```

Comment:
Let's add a [`helm_repository`](https://www.terraform.io/docs/providers/helm/repository.html) resource for this repo.

Reply:
Done